src/main/java/hudson/plugins/active_directory/CacheConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package hudson.plugins.active_directory;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+
+/**
+ * Cache for Users and Groups
+ */
+public class CacheConfiguration<K,V,E extends Exception> {
+    public final int size;
+    public final int ttl;
```

Comment:
`private`

If you ever find you have written a public field that is not `final static` then you should stop and think "why?"